doc/src/tutorial.xml

```diff
@@ -280,8 +280,8 @@ top/
       in their requirements, too.  Of course, any project can add
       include paths to those specified by its parents. <footnote>
       <para>Many
-      features will be overridden,
-      rather than added-to, in subprojects.  See <xref
+      features will be overridden in,
+      rather than added-to, subprojects.  See <xref
```

Comment:
AMDG

On 04/29/2016 10:44 AM, Edward Diener wrote:
>> @@ -280,8 +280,8 @@ top/
>>        in their requirements, too.  Of course, any project can add
>>        include paths to those specified by its parents. <footnote>
>>        <para>Many
>> -      features will be overridden,
>> -      rather than added-to, in subprojects.  See <xref
>> +      features will be overridden in,
>> +      rather than added-to, subprojects.  See <xref
> 
> I disagree. The phrase 'added-to in subprojects' cannot be correct English while the phrase 'added-to subprojects' is the correct terminology.
> 

It's the features that are being added-to, not the subproject.

In Christ,
Steven Watanabe